hegelai/prompttools
# Refactor PromptTest module

1. Change prompttest API
* Previously, we annotated an evaluation function. Now, we annotate a prompt provider function. This seems like a more natural way to test prompts.
* `expected` argument for semantic similarity is still somewhat hacky.
* See `examples/prompttest/test_openai_chat.py` or `examples/prompttest/test_huggingface_hub.py` for examples.

2. Add `evaluate` functions to utilities
